modules/client-configuration/pages/clients-ubuntu-old.adoc

```diff
@@ -22,282 +22,133 @@ However, the root user on {ubuntu} is disabled by default, so to use bootstrappi
 
 
 
-== Prepare to Register
-
-Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
+== Add Software Channels
 
+Before you register {ubuntu} clients to your {productname} Server, check that you have the {ubuntu} product enabled, and the required channels are fully synchronized.
 
 ifeval::[{suma-content} == true]
-.Procedure: Adding Client Tools Channels
-
-Before you begin, ensure you have the {ubuntu} product enabled, and have synchronized the {ubuntu} channels for {scc}:
-
-You can do this from {webui} or using command prompt, at your choice.
-
-For {ubuntu} 16.04:
 
-* From the {webui}, add [systemitem]``Ubuntu 16.04`` and [systemitem]``SUSE Linux Enterprise Client Tools Ubuntu 1604 amd64``.
-* From the command prompt, add [systemitem]``ubuntu-16.04-pool-amd64`` and [systemitem]``ubuntu-16.04-suse-manager-tools-amd64``.
+The channels you need for this procedure are:
 
-For {ubuntu} 18.04:
+[[ubuntu-old-channels-wizard]]
+[cols="1,1,1", options="header"]
+.Ubuntu Channels - WebUI
+|===
+| OS Version     | Base Channel | Updates Channel
+| {ubuntu} 16.04 | Ubuntu 16.04 | ubuntu-16.04-pool for amd64
+| {ubuntu} 18.04 | Ubuntu 18.04 | ubuntu-18.04-pool for amd64
```

Comment:
Wrong:

What you listed as "Base Channel" is the product name and what is listed as "Updates Channel" is the "Base Channel"